README.md

```diff
@@ -1 +1,189 @@
+
+### Setup
+
+Clone this repository to your desired folder on your local machine
+
+### Install
+
+The projects incorporate some linters check, to get them running on your local system follow these below;
+
+1. `.github/workflows/linters.yml` is already included in project, this is used by Github to run tests on the repo.
+2. To install Webpack packages and dependencies
+
+- `npm init -y` initalize the project.
+- `npm i` installs all dependencies listed in the projects's package.json
+
+- `npm install webpack webpack-cli --save-dev` installs webpack
+- `npm install --save-dev html-webpack-plugin` installs webpack plugin
+- `npm install --save-dev style-loader css-loader` installs assets loaders
+- `npm install --save-dev webpack-dev-server` installs webpack server
+
+3. To install Stylelint linters run:
+
+- `npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x`
+- `npx stylelint "\*_/_.{css,scss}" --fix" to run the test.
+
+4. To Install ESLint run:
+
+- `npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x`
+- `npx eslint . --fix` to run the test
+
+5. To install Webhint linters run:
+
+- `npm install --save-dev hint@7.x`
+- `npx hint .` to run the test.
+
+### Usage
+
+To run the project, clone the repository to your local machine, install dependencies and launch through webpack Server on VScode
+
+### Run tests
+
+Follow the procedures under [Install](#install) to run some of the included linter tests.
+
+### Deployment
+
+You can deploy this project using:
+
+- [Github Page](https://pages.github.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Damilare Adepoju**
+
+- GitHub: [@githubhandle](https://github.com/adamilare)
+- Twitter: [@twitterhandle](https://twitter.com/mailtodare)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/adamilare)
+- GitUp Page: [My Page](https://adamilare.github.io/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcomed!
+
+Feel free to check the [issues page](https://github.com/adamilare/leaderboard/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a thumb up.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse Inc. for providing template and other guidelines for this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+N/A
+
```

Comment:
[OPTIONAL] this section is an optional section hence it can be removed in cases where you do not have questions to answer. It would be better to remove it instead of putting N/A because users would always have a question about your app.